TEST_GUIDE.md

```diff
@@ -6,9 +6,12 @@ The full list of builds is on the [releases](https://github.com/AllYarnsAreBeaut
 
 If you're just interested in poking around, choose the most recent build.
 
-When you click on a build you should see a list of files under the **Assets** section. Click on the correct file for your operating system. If you're on a mac, you want the **.dmg**, if you're on windows you want the **.exe**, if you're on Linux you'll have to build from source.
+When you click on a build you should see a list of files under the **Assets** section. Click on the correct file for your operating system. If you're on a mac, you want the **.dmg**, if you're on windows you want the **.exe**. Open the file you downloaded and install it the way you normally would. If you're on Linux you'll have to build from source.
+
+
+### Extra Linux setup
```

Comment:
I think it's an always an issue if you build from source, no matter on which OS

Reply:
Yeah it's an issue whenever you build from source, independent on OS, but for testing, building from source is only needed for Linux (Windows gets an .exe and Mac gets a .dmg).